### DISCUSSION TITLE
博客评论排版异常

### DISCUSSION THREAD

**Discussion by tangluya** [walinejs/waline#1920] [Q&A] [answered]
大佬们好，我博客使用的是hexo主题：[oranges](https://github.com/zchengsite/hexo-theme-oranges/)
我这个问题已经在oranges主题的issues中提出（[waline的一点问题](https://github.com/zchengsite/hexo-theme-oranges/issues/91)），但是作者近期似乎长时间不在线
我不懂代码，不得已尝试来这里寻求帮助，不知道有没有兄弟也遇到这种问题

我博客waline异常的例子：https://blog.tanglu.me/use-rclone-to-upload-torrents-when-downloaded/

![image](https://user-images.githubusercontent.com/105842721/229670081-fd42f83e-4f49-4797-ab2d-cab91a28c17a.png)

**Comment by Mister-Hope** [comment, marked as answer]
<img width="1027" alt="image" src="https://user-images.githubusercontent.com/33315834/229738718-17ef53ba-9419-4aa4-98a8-f5327eac2414.png">

The problem is that you are using latest style file with v2, but using a very old version of client 2.5.1, I update the style file cdn link manually to 2.5.1 and you can see it's working well. Either you use latest version of client, or you fix the style file to 2.5.1

**Comment by tangluya** [comment]
@Mister-Hope  Thank you for your kindless reply,sir.I don't know how to fix the hexo_theme_oranges's waline client  .So I think I can only update the style file cdn link to 2.5.1,but I didn't know how to set the style cdn link. Do you know how I should probably set it up?
My current style cdn link is this:
```
comments:

    waline:
      enable: true
      url: https://cdn.jsdelivr.net/npm/@waline/client@2.5.1/dist/legacy.min.js
```

**Reply by Mister-Hope** [reply]
Delete `.5.1`

**Comment by tangluya** [comment]
![image](https://user-images.githubusercontent.com/105842721/229754018-dea53091-5023-4320-976a-f85a5bf0036c.png)

**Comment by tangluya** [comment]
I'm guessing it's a fix for this [file](https://github.com/zchengsite/hexo-theme-oranges/search?q=unpkg), but I don't know the exact settings
